draft-foudil-securitytxt.md

```diff
@@ -644,6 +736,8 @@ of DNS-stored encryption keys (#28 and #94)
 ## Since draft-foudil-securitytxt-05
 - Changing HTTPS to MUST (#55)
 - Adding language recommending encryption for email reports (#134)
+- Added language handling redirects (#143)
+- Expanded security considerations section (#30, #73, #103, #112)
```

Comment:
Not sure if it's notable enough but you also fixed some typos with "fall back" -> "fallback". Spelling corrections were mentioned in the history update for the "since -02" entry.